hooks/usePoH.ts

```diff
@@ -29,16 +32,18 @@ export const usePoH = (
     } catch (error) {
       // eslint-disable-next-line no-console
       console.error('Error fetching Proof of Humanity', error);
+    } finally {
+      setFetching(false);
     }
   }, [address]);
 
   useEffect(() => {
-    getPOH();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [address]);
+    getPoH();
+  }, [getPoH]);
```

Comment:
when i put gePoH into the dependencies array, the hook kept rerendering the quest chain page, so all the actions were blocked and the tab froze. i went to look if it works, but noticed that there's another error that's popping - `GraphQLError: indexing_error at rehydrateGraphQlError`
could you leave the dependency as it was before, to not freeze the quest chain page if the query starts working again?

Reply:
@vidvidvid this hook wasn't being used in the quest chain page but `usePoHs` was.. So i took a look at the code and added a fix in [26a0c80](https://github.com/quest-chains/dapp/pull/175/commits/26a0c80b42012b729eca9c49d39933d9ce102096)